types/watecco/triphaso/variables.schema.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$id": "https://akenza.io/watecco/triphaso/variables.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Phase variables",
+  "processingType": "uplink_decoder",
+  "topic": "variables",
```

Comment:
what's the difference to the topic phase_variables?

Reply:
I splitted them up